hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1519,6 +1519,14 @@
       executed since last report. Unit could be defined with
       postfix (ns,ms,s,m,h,d)</description>
   </property>
+  <property>
+    <name>hdds.command.delete.worker.interval</name>
+    <value>2000</value>
+    <tag>OZONE, DATANODE, MANAGEMENT</tag>
+    <description>The interval between DeleteCmdWorker execution of delete
+      commands. The unit is milliseconds.
+    </description>
+  </property>
```

Comment:
Please add the new config property to `DatanodeConfiguration` instead of `ozone-default.xml`.

There you'll find other properties related to block deletion, e.g.:

https://github.com/apache/ozone/blob/366c2fb57ae15acbb4c357086328ddec2faeb4f3/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java#L172-L185

Please use the same naming convention for the new key, e.g. `block.delete.command.worker.interval`.  (Prefix `hdds.datanode.` is automatically added.)